pkg/kubectl/cmd/get/get.go

```diff
@@ -229,8 +229,9 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 					printFlags.UseOpenAPIColumns(apiSchema, mapping)
 				}
 			}
+			printFlags.SetKind(mapping.GroupVersionKind.GroupKind())
 			if resource.MultipleTypesRequested(args) {
```

Comment:
can this be hoisted out of the ToPrinter() loop and set just once?

Reply:
Done.